### DISCUSSION TITLE
Replicating Empirical Bayesian Kriging functionality

### DISCUSSION THREAD

**Discussion by goriliukasbuxton** [GeoStat-Framework/PyKrige#249] [Q&A]
Has anyone done similar work like this tool? to get most optimal parameters for Kriging
https://pro.arcgis.com/en/pro-app/latest/help/analysis/geostatistical-analyst/what-is-empirical-bayesian-kriging-.htm

**Comment by MuellerSeb** [comment]
Isn't this the way sklearn fits the gaussian processes kernels?
https://scikit-learn.org/stable/modules/gaussian_process.html
